docs/platform/block-storage/how-to-use-block-storage-with-your-linode-new-manager/index.md

```diff
@@ -0,0 +1,184 @@
+2.  You'll need to create a filesystem in your new volume. If your Linode is not already running, boot then SSH into your Linode and execute the following command, where `FILE_SYSTEM_PATH` is your volume's file system path:
+
+        mkfs.ext4 FILE_SYSTEM_PATH
+
+3.  Once the volume has a filesystem, you can create a mountpoint for it:
+
+        mkdir /mnt/BlockStorage1
+
+4.  You can then mount the new volume:
+
+        mount FILE_SYSTEM_PATH /mnt/BlockStorage1
+
+5.  If you want to mount the new volume automatically every time your Linode boots, you'll want to add the following line to your **/etc/fstab** file:
+
+        FILE_SYSTEM_PATH /mnt/BlockStorage1 ext4 defaults 0 2
+
+### Attach a Volume from Your Account's Volume List
+
+1.  Click on the **Volumes** page of the Linode Manager to see your account's volume list:
+
+    [![View your available Volumes](bs-cloud-volumes-list-small.png "View your available Volumes.")](bs-cloud-volumes-list.png)
+
+2.  Click the menu option (three dots) for the volume you want to attach to a Linode and select **Attach**:
+
+    [![Open volume menu.](bs-cloud-attach-volume-small.png)](bs-cloud-attach-volume.png)
+
+3.  Select the label of the Linode you want to attach the volume to from the dropdown menu, then click **Save**:
+
+    [![Attach a Volume to a Linode](bs-cloud-attach-volume-menu.png "Attach a Volume to a Linode")](bs-cloud-attach-volume-menu.png)
+
+    {{< note >}}
+The Linodes available in this dropdown menu all share the same region as your volume.
+{{< /note >}}
+
+
+4.  You'll need to create a filesystem in your new volume. If your Linode is not already running, boot then SSH into your Linode and execute the following command, where `FILE_SYSTEM_PATH` is your volume's file system path:
+
+        mkfs.ext4 FILE_SYSTEM_PATH
+
+5.  Once the volume has a filesystem, you can create a mountpoint for it:
+
+        mkdir /mnt/BlockStorage1
+
+6.  You can then mount the new volume, where `FILE_SYSTEM_PATH` is your volume's file system path:
+
+        mount FILE_SYSTEM_PATH /mnt/BlockStorage1
+
+7.  If you want to mount the new volume automatically every time your Linode boots, you'll want to add the following line to your **/etc/fstab** file:
+
+        FILE_SYSTEM_PATH /mnt/BlockStorage1
+
+## How to Detach a Block Storage Volume from a Linode
+
+1.  Go back to the page of the Linode which the volume is attached to. Shut down the Linode.
+
+2.  When the Linode is powered off, click on the **Volumes** tab, then click **Detach** under the volume's menu (three dots):
+
+    [![Detach a Volume from a Linode from the Volume menu.](bs-cloud-detach-volume-small.png "Detach a Volume from a Linode from the Volume menu.")](bs-cloud-detach-volume.png)
+
+3.  A confirmation screen appears and explains that the volume will be detached from the Linode. Click **Detach** to confirm:
+
+    [![Linode Manager detach volume confirmation](bs-cloud-detach-volume-confirm.png)](bs-cloud-detach-volume-confirm.png)
```

Comment:
Styles for this modal have been updated recently, the confirm button is now red. Not sure if this is important enough to justify changing the screenshot (the theme will continue to be adjusted over the next few months)